Bio/Nexus/Nexus.py

```diff
@@ -593,9 +593,7 @@ def __init__(self, line, title):
                         for n in range(len(options))
                         if options[n] == "=" and n != 0 and n != len(options)
                     ]
-                    indices = []
-                    for sl in valued_indices:
-                        indices.extend(sl)
+                    indices = [index for sl in valued_indices for index in sl]
```

Comment:
I feel there ought to be another clearer way to express this, perhaps using itertools.chain?

Reply:
I guess it is a matter of taste. When I see `itertools.chain`, I first have to look up in the documentation to remind myself what it does.